classes/class4/homework/package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Did you mean to put your class 3 homework in your class 4 folder? Make sure you are careful of that in the future. Took us a minute to find the right application.